command/format/diff_test.go

```diff
@@ -1432,6 +1679,72 @@ func TestResourceChange_primitiveList(t *testing.T) {
           + (known after apply),
           + (known after apply),
             "cccc",
+            "dddd",
+            "eeee",
+        ]
+    }
+`,
+		},
+	}
+	runTestCases(t, testCases)
+}
+
+func TestResourceChange_primitiveTuple(t *testing.T) {
```

Comment:
This new test case added for completeness, although it is covering basically the same code as `primitiveList`.